frontend/workflows/projectCatalog/src/index.tsx

```diff
@@ -12,7 +12,7 @@ type DetailCard = CatalogDetailsCard | typeof DynamicCard | typeof MetaCard;
 export interface ProjectConfigProps {
   title: string;
   path: string;
-  onError: (error: ClutchError) => void;
+  onError?: (error: ClutchError) => void;
```

Comment:
Ah yeah, we might want to have an internal version of our props for this purpose since even if users set an onError prop we would overwrite it with out own callback

Reply:
Thinking...should we even worry about a top level error or keep it confined into the individualized components? Then it can be handled as they see fit. If not I can just remove this logic entirely.

Reply:
I think having consistent error handling is nice but I see the added complexity here. If each component / page were to define it's own error handler we might end up with inconsistencies in the experience between tabs

Reply:
Alright, I modified it to keep the same props but will check if the child has already set an `onError` handler and call that before setting the error